code/modules/antagonists/traitor/contractor/datums/objective_contract.dm

```diff
@@ -17,7 +17,6 @@
 		EXTRACTION_DIFFICULTY_EASY = list(
 			// Rooms
 			"Alternate Construction Area",
-			"Barber Shop",
```

Comment:
You don't actually remove this area in the PR, please keep this

Reply:
Fixed, sorry!